osu.Game/Screens/SelectV2/SongSelectV2.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Screens;
+using osu.Game.Overlays;
+using osu.Game.Overlays.Mods;
+using osu.Game.Screens.Menu;
+using osu.Game.Screens.Play;
+using osu.Game.Screens.SelectV2.Footer;
+using osuTK;
+
+namespace osu.Game.Screens.SelectV2
+{
+    public partial class SongSelectV2 : OsuScreen
```

Comment:
Probably worth adding an xmldoc mentioning that this is a WIP and will gradually be built up to a usable level.